extensions/@shopgate-theme-config/extension/mockCatalog2Data.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-param-reassign */
+const mockEffectiveDates = require('./mockEffectiveDates');
+const mockMapPrice = require('./mockMapPrice');
+const mockMinMaxOrder = require('./mockMinMaxOrder');
+const mockNotFoundProduct = require('./mockNotFoundProduct');
+
+/**
+ * Returns default configuration if request failed.
+ * @param {SDKContext} context context
+ * @param {Object} input input
+ * @returns {Promise<{products: Object[]}>}
+ */
+module.exports = async (context, { products = [] }) => {
+  if (!products.length || products.length === 1) {
```

Comment:
The target branch has a backend config now to enable or disable mock data. Please consider using this config here.
You might need to fix some merge conflicts, first